sale_order_lot_selection/models/stock_lot.py

```
@@ -0,0 +1,48 @@
+from odoo.exceptions import AccessError
+
+
+class StockLot(models.Model):
+    _inherit = "stock.lot"
+
+    def action_add_to_sale_order(self):
+        """Open the wizard to add lots to a sale order."""
+        self.ensure_one()
+        return self.action_generate_sale_order()
+
+    def action_generate_sale_order(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `action_generate_sale_order`

(ANN201)